libs/block-content/website/src/lib/teaser/teaser-grid-block.tsx

```diff
@@ -48,6 +48,8 @@ export const TeaserGridBlock = ({numColumns, teasers, className}: BuilderTeaserG
         <Teaser
           key={index}
           teaser={teaser}
+          index={index}
```

Comment:
Should maybe done inside `TeaserGridBlock` itself to select the correct `Teaser` but fine IMO for now.